vcpkg.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Do we still support vcpkg?

Reply:
No we dont but sometimes I add it back in to get debug builds of boost which helps with debugging fibers issues. Will remove